src/EnergyPlus/HVACVariableRefrigerantFlow.hh

```diff
@@ -700,6 +700,10 @@ namespace HVACVariableRefrigerantFlow {
         int ATMixerSecNode;              // secondary air inlet node number for the air terminal mixer
         int ATMixerOutNode;              // outlet air node number for the air terminal mixer
         bool firstPass;                  // used to reset global sizing data
+        Real64 coilInNodeT;              // coil inlet node temp at full flow (C)
```

Comment:
new variables to hold data in order to eliminate the problem with flow mismatch from inlet to outlet.